resources/function-controller/README.md

```diff
@@ -58,11 +54,11 @@ Follow the steps to install the chart:
                  --set global.ingress.domainName="${DOMAIN_NAME}" \
                  --tls
     ```
+
 ### Run the first function
 
-Currently, there is no UI support for the new Function Controller. To use it, follow these steps:
-1. Execute steps 4-6 from [these](https://github.com/kyma-project/kyma/blob/master/components/function-controller/README.md#installation) instructions.
-2. Run your first function using [these](https://github.com/kyma-project/kyma/blob/master/components/function-controller/README.md#create-a-sample-hello-world-function) steps.
+Currently, there is no UI support for the new Function Controller. Follow [these](https://github.com/kyma-project/kyma/blob/master/components/function-controller/README.md#create-a-sample-hello-world-function) instructions to deploy function in `serverless-system
```

Comment:
```suggestion
Currently, there is no UI support for the Function Controller. Follow [these](https://github.com/kyma-project/kyma/blob/master/components/function-controller/README.md#create-a-sample-hello-world-function) instructions to deploy a function in the Namespace you specified during the chart installation (step 3).
```

Reply:
Is `specify` ok for you? `determine` is more "to establish exactly by research or calculation".

Reply:
It's ok! Thanks